forest/src/cli/config_cmd.rs

```diff
@@ -42,6 +42,7 @@ mod tests {
 
         let actual_config: Config = toml::from_str(std::str::from_utf8(sink.buffer()).unwrap())
             .expect("Invalid configuration!");
-        assert_eq!(actual_config, expected_config);
+
+        assert!(expected_config == actual_config);
```

Comment:
Hm, why?

Reply:
This is to avoid to implement `Debug`, needed by `assert_eq`...

Reply:
Makes sense. I added the `Debug` in the linked PR as well.